client/tests/unit/example.spec.js

```diff
@@ -0,0 +1,5 @@
+describe('Sample Test', () => {
```

Comment:
I'm not sure we need this particular test file. Can we round out a true test for one of the pages? or remove this example file?